library/Makefile

```diff
@@ -91,9 +104,7 @@ shared: libmbedcrypto.$(DLEXT) libmbedx509.$(DLEXT) libmbedtls.$(DLEXT)
 # tls
 libmbedtls.a: $(OBJS_TLS)
 	echo "  AR    $@"
-	$(AR) -rc $@ $(OBJS_TLS)
-	echo "  RL    $@"
-	$(AR) -s $@
+	$(AR) $@ $(OBJS_TLS)
```

Comment:
I have already tried all these options to find compatibility with ```ar```. These options are incompatible:
```
$iarchive --create --replace
Command line error: Illegal combination: --replace/-r and --create
```